backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/manufacturing/job/service/ManufacturingJobDefaultFilter.java

```diff
@@ -2,5 +2,5 @@
 
 public enum ManufacturingJobDefaultFilter
 {
-	UserPlant, TodayDatePromised,
+	UserPlant, TodayDateStartSchedule,
```

Comment:
what shall happen with the systems where the sysconfig contains the `TodayDatePromised` value?

Reply:
thx, added update statement in `backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5727040_sys_gh18288_update_sysconfig_mobileuiDefaultFilters.sql`